Source/DiabloUI/diabloui.cpp

```diff
@@ -510,7 +510,7 @@ void LoadHeros()
 		Art portrait;
 		char portraitPath[18];
 		sprintf(portraitPath, "ui_art\\hero%i.pcx", i);
-		LoadArt(portraitPath, &portrait);
+		LoadArt(portraitPath, &portrait, 1, nullptr, nullptr, false);

```

Comment:
This is the only case where the `showError` parameter is used.
How can `LoadArt` ever succeed in this case (`heroXYZ.pcx` exists)?
Is this only possible when loading a custom assets/mpqs?

Reply:
Yes. This is for loading optional assets so you don't want to throw an error when they don't exist.